service/mapper/pathings_test.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package mapper
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/ledger/common/hash"
+	"github.com/onflow/flow-go/ledger/complete/mtrie/node"
+	"github.com/onflow/flow-go/ledger/complete/mtrie/trie"
+)
+
+// Common test variables used for both pathings_test.go and transitions_test.go.
+var (
+	testPath1 = ledger.Path{0xaa, 0xc5, 0x13, 0xeb, 0x1a, 0x04, 0x57, 0x70, 0x0a, 0xc3, 0xfa, 0x8d, 0x29, 0x25, 0x13, 0xe1, 0xaa, 0xc5, 0x13, 0xeb, 0x1a, 0x04, 0x57, 0x70, 0x0a, 0xc3, 0xfa, 0x8d, 0x29, 0x25, 0x13, 0xe1}
+	testPath2 = ledger.Path{0xd5, 0x08, 0x44, 0x13, 0xdb, 0xe5, 0x2b, 0xd2, 0x3a, 0x66, 0x7f, 0xc4, 0x08, 0xe0, 0x54, 0x60, 0xd5, 0x08, 0x44, 0x13, 0xdb, 0xe5, 0x2b, 0xd2, 0x3a, 0x66, 0x7f, 0xc4, 0x08, 0xe0, 0x54, 0x60}
+	testPath3 = ledger.Path{0x60, 0x0a, 0xd8, 0xa4, 0xf1, 0x6b, 0xce, 0x2e, 0x57, 0x59, 0xfd, 0x6e, 0x45, 0xcf, 0xa9, 0xa0, 0x60, 0x0a, 0xd8, 0xa4, 0xf1, 0x6b, 0xce, 0x2e, 0x57, 0x59, 0xfd, 0x6e, 0x45, 0xcf, 0xa9, 0xa0}
+	testPath4 = ledger.Path{0xa5, 0x68, 0x7b, 0x2d, 0x95, 0x18, 0x7b, 0xc7, 0xce, 0xd0, 0xe1, 0x02, 0xd6, 0xce, 0xfe, 0x93, 0xa5, 0x68, 0x7b, 0x2d, 0x95, 0x18, 0x7b, 0xc7, 0xce, 0xd0, 0xe1, 0x02, 0xd6, 0xce, 0xfe, 0x93}
+	testPath5 = ledger.Path{0x60, 0x0a, 0xd8, 0xa4, 0xf1, 0x6b, 0xce, 0x2e, 0x57, 0x59, 0xfd, 0x6e, 0x45, 0xcf, 0xa9, 0xa0, 0x60, 0x0a, 0xd8, 0xa4, 0xf1, 0x6b, 0xce, 0x2e, 0x57, 0x59, 0xfd, 0x6e, 0x45, 0xcf, 0xa9, 0xa0}
+	testPath6 = ledger.Path{0xfd, 0x84, 0xc0, 0xa7, 0xb2, 0x35, 0xc9, 0x89, 0xc1, 0x8e, 0x6a, 0xa2, 0x69, 0x04, 0xfe, 0xba, 0xfd, 0x84, 0xc0, 0xa7, 0xb2, 0x35, 0xc9, 0x89, 0xc1, 0x8e, 0x6a, 0xa2, 0x69, 0x04, 0xfe, 0xba}
```

Comment:
I like to put these into groups of 8, so 4 lines of 8 values:

```
testPath1 = ledger.Path {
  ...
  ...
  ...
  ...
}
```

 A lot more readable imo.